app/views/bgeigie_imports/_status.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= stylesheet_link_tag "application.css" %>
```

Comment:
I think we already [doing](https://github.com/Safecast/safecastapi/blob/master/app/views/layouts/application.html.erb#L13) it